Mutagen.Bethesda.Analyzers.Skyrim/Record/Dialog/Responses/InconsistentCharactersAnalyzer.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Mutagen.Bethesda.Analyzers.SDK.Analyzers;
+using Mutagen.Bethesda.Analyzers.SDK.Results;
+using Mutagen.Bethesda.Analyzers.SDK.Topics;
+using Mutagen.Bethesda.Skyrim;
+using Noggog;
+namespace Mutagen.Bethesda.Analyzers.Skyrim.Record.Dialog.Responses;
+
+public class InconsistentCharactersAnalyzer : IIsolatedRecordAnalyzer<IDialogResponsesGetter>
+{
+    public static readonly TopicDefinition<string, string> PromptInconsistentCharacters = MutagenTopicBuilder.DevelopmentTopic(
+            "Prompt Has Inconsistent Characters",
+            Severity.Suggestion)
+        .WithFormatting<string, string>("Response {0} contains characters {1} which are not usually used in dialog");
+
+    public static readonly TopicDefinition<string, string> ResponseInconsistentCharacters = MutagenTopicBuilder.DevelopmentTopic(
+            "Response Has Inconsistent Characters",
+            Severity.Suggestion)
+        .WithFormatting<string, string>("Response {0} contains characters {1} which are not usually used in dialog");
```

Comment:
Why are these not allowed?  Does it cause issues?

Reply:
It's a bit opinionated because these are characters that my project uses to mark placeholder lines and notes how dialogue should behave which I would like to report when they're still in game. In general though, these characters are never used in Skyrim, only in unfinished content and one regular case which is an anomaly. Using it would definitely be inconsistent with Skyrim's typical writing.

Here are two examples of cut content where it's used
> Response Join the forces mustering at the [CAPITAL] (infos conditions on location go here)
> Response Join the forces mustering outside of [CAPITAL]. (Infos conditioned on Capital location go here)